osmosis-1/osmosis.zone_assets.json

```diff
@@ -4065,6 +4065,15 @@
       "osmosis_verified": false,
       "listing_date_time_utc": "2024-06-25T12:00:00Z",
       "_comment": "N43 FanToken $N43"
+    },
+    {
+      "chain_name": "terra",
+      "base_denom": "ibc/46579C587A0B8CF8B0A1FF6B0EFA2082F11876578E47FC81A9CAAD31F424AF98",
```

Comment:
base_denom would be cw20:terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2, not the ibc hash

Reply:
Sorry. So base denom means always the last part of the full ibc path? Because in that case all the ibc assets should have the base denom of their home network... Like "uluna" for example. But for some of these assets the base denom doesn't match the last part in the ibc path....

Anyway, corrected and thanks for the review.